.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "airbnb-base",
+  "rules": {
+    "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/test/**/*.js"]}],
```

Comment:
This is a pretty handy way of turning off the no-extraneous-dependencies rule for only test-related files